keyboards/und/pcm-t-k0-windows.xml

```diff
@@ -0,0 +1,327 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboard
+  SYSTEM '../dtd/ldmlKeyboard.dtd'>
+<keyboard locale="pcm-t-k0-windows">
+	<version number="$Revision$" platform="10"/>
+	<names>
+		<name value="Naijíriá Píjin"/>
```

Comment:
```suggestion
		<!-- also known as Naijíriá Píjin -->
```

This is the problem, `<name/>` is duplicated, it may only be a singleton.  https://unicode.org/reports/tr35/tr35-keyboards.html#Element_name

Localization of these names is a separate discussion… maybe worth a separate issue?